app/posts/register-trainee-teachers/2023-01-31-asking-providers-to-sign-off-trainee-data-for-the-previous-academic-year.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Asking providers to sign off trainee data for the previous academic year
+description: We created a way for providers to confirm which trainees qualified, withdrew and deferred in the previous academic year.
+date: 2023-01-31
+---
+
+In January each year, training providers need to check and sign off their trainee data from the previous academic year. For example, in January 2023 they need to check the data for the 2021 to 2022 academic year.
+
+This is so that the data can be included in the annual [initial teacher training performance profiles publication](https://explore-education-statistics.service.gov.uk/find-statistics/initial-teacher-training-performance-profiles). This gives information about the outcomes for trainee teachers in England, including the proportion of trainees who:
+
+- were in their final year and gained qualified teacher status (QTS) or early years teacher status (EYTS)
+- got a job in a state school within 16 months of gaining QTS
+
+We decided to ask providers to check data for all their trainees for the 2021 to 2022 academic year, not just those who finished their courses in that year. This is not necessary for the performance profiles but will help make sure that trainee data is accurate.
+
+## What we changed
+
+We added:
+
+- guidance for users
+- a report containing the data which users need to check
```

Comment:
```suggestion
- a report containing the data which users need to check
- a web form for providers to use to sign off their data
```

Reply:
I know you mention this below as part of guidance, but I'd also consider it a distinct thing.

Reply:
I've added it to the first bullet (guidance) instead of having it as a bullet in its own right. I think that works better as the information about it is in the guidance section rather than under its own subheading.